src/lexer.coffee

```diff
@@ -1178,6 +1178,12 @@ CSX_IDENTIFIER = /// ^
   ( (?: (?!\s)[\.\-$\w\x7f-\uffff] )+ ) # Like `IDENTIFIER`, but includes `-`s and `.`s.
 ///
 
+# Fragment: <></>
+CSX_FRAGMENT_IDENTIFIER = /// ^
+  (?![\d<])   # Must not start with number or `<`
+  (.{0}(?=>)) # Ends immediately with '>.
+///
```

Comment:
I think this could become:

```coffee
CSX_FRAGMENT_IDENTIFIER = /^((?=>))/ # Ends immediately with `>`.
```

You’re really only looking for the next character to be `>`, and capturing a zero-length string so that the regex `exec` matches. The stuff about “must not start with a number or `<`“ refers to the identifier inside a normal CSX tag, e.g. the `div` in `<div>`. But by definition a fragment has no identifier, so we don’t need to validate one.